tests/app/dao/test_organisation_dao.py

```diff
@@ -1,10 +1,11 @@
-import datetime
-import uuid
+"""
+TODO - This file has multiple uses of using the notify_db_session fixture only for the
+side-effect of clearing tables after the test.
```

Comment:
This is a code smell.

Reply:
see above re TODOs and the need for a corresponding ticket and that ticket number added to the comment.

Reply:
#1106